### PR TITLE
Issue #2878: Add back the contextual filter input option from arg().

### PR DESCRIPTION
**Pull request by jenlampton**
https://github.com/backdrop/backdrop-issues/issues/2878

### CODE REVIEW

File: core/modules/views/plugins/views_plugin_display_block.inc [hunk 1]
```diff
@@ -306,7 +327,7 @@ class views_plugin_display_block extends views_plugin_display {
           $form['argument_input'][$id]['context_optional'] = array(
             '#type' => 'checkbox',
             '#title' => t('Context is optional'),
-            '#description' => t('This context need not be present for the block to function. If you plan to use this, ensure that the argument handler can handle empty values gracefully.'),
+            '#description' => t('This context need not be present for the block to function. If you plan to use this, ensure that the contextual filter handler can handle empty values gracefully.'),
```
Comment (klonos) [inline core/modules/views/plugins/views_plugin_display_block.inc:330]:
```suggestion
            '#description' => t('This context is not required for the block to function. If you plan to use this, ensure that the contextual filter handler can handle empty values gracefully.'),
```

...a little less poetic :)

File: core/modules/views/plugins/views_plugin_display_block.inc [hunk 2]
```diff
@@ -318,7 +339,7 @@ class views_plugin_display_block extends views_plugin_display {
           $form['argument_input'][$id]['fixed'] = array(
             '#type' => 'textfield',
             '#title' => t('Fixed argument'),
-            '#description' => t('If "Fixed" is selected, what to use as an argument.'),
+            '#description' => t('This value will be set permanently as the filter value.'),
```
Comment (klonos) [inline core/modules/views/plugins/views_plugin_display_block.inc:342]:
```suggestion
            '#description' => t('This value will be permanently set as the filter value.'),
```

**Comment by jenlampton** [comment]
yes, thank you @klonos !

File: core/modules/views/plugins/views_plugin_display_block.inc [hunk 3]
```diff
@@ -330,7 +351,7 @@ class views_plugin_display_block extends views_plugin_display {
           $form['argument_input'][$id]['label'] = array(
             '#type' => 'textfield',
             '#title' => t('Label'),
-            '#description' => t('If this argument is presented to the administrator configuring a layout, what label to apply to it.'),
+            '#description' => t('Label to apply to the filter when it is presented to the person configuring a layout.'),
```
Comment (klonos) [inline core/modules/views/plugins/views_plugin_display_block.inc:354]:
```suggestion
            '#description' => t('The admin label of the filter when configuring a layout.'),
```

**Comment by jenlampton** [comment]
new sandbox plz!

**Comment by opi** [comment]
untested, but code looks good.

**Comment by quicksketch** [comment]
Looks good! Merged into 1.x for 1.12.0.

**Pull request merged**